truss-chains/truss_chains/__init__.py

```diff
@@ -39,3 +38,22 @@
 )
 from truss_chains.stub import StubBase
 from truss_chains.utils import make_abs_path_here
+
+__all__ = [
```

Comment:
This explicit list seems like a maintenance drag without any actual value. Given that `__init__` files are somewhat special and shouldn't contain implementation code, could we use a module-global ignore to silence that error?

Are there any benefits to using `__all__`?

Reply:
The main benefit is ensuring that wildcard imports don't unnecessarily import everything from the `__init__.py`. It's also generally the best way of describing a module's public API per [PEP 0008](https://peps.python.org/pep-0008/).

If you all don't want to maintain them though, I can disable the rule just for `__init__.py` files.

Reply:
I'm weighing the trade-off between following pep8 more closely and the maintenance burden.

I'm leaning towards deviating from pep8:
* Wildcard imports should not be done, even if pep8 doesn't forbid them.
* Keeping/maintaining that list seems redundant.
* If any symbols are imported/defined that are not meant to be in the module's API, they should be `del`eted.

So please disable the inspection for init-files.

Reply:
Sounds good. Will do!

Reply:
https://github.com/basetenlabs/truss/pull/971/commits/7c48b242ba095552599edf50e5bd4345d772afa7